src/relay/transforms/partial_eval.cc

```diff
@@ -776,8 +777,9 @@ class PartialEvaluator : public ExprFunctor<PStatic(const Expr& e, LetList* ll)>
 
   Func VisitFuncStatic(const Function& func, const Expr& var) {
     CHECK(IsAtomic(var));
+    // todo: figure out primitive semantic
     if (func->HasNonzeroAttr(attr::kPrimitive)) {
-      return ConstEvaluateFunc(func);
+      // return ConstEvaluateFunc(func);
```

Comment:
Isn't this dead code if you don't do anything in the body?

Reply:
this is debug stuff. i havent clean up yet